app/controllers/crate/version.js

```diff
@@ -34,7 +34,7 @@ export default Controller.extend({
     return this.get('crate.owner_user').findBy('id', this.get('session.currentUser.id'));
   }),
   notYankedOrIsOwner: computed('model', 'crate.owner_user', 'session.currentUser.id', function () {
-    return !this.get('model').yanked || this.get('crate.owner_user').findBy('id', this.get('session.currentUser.id'));
+    return !this.model.yanked || this.get('crate.owner_user').findBy('id', this.get('session.currentUser.id'));
```

Comment:
I tried tracing the logic and I think `this.model` comes from Ember Data so we might want to leave the `get` here.

Reply:
removed